### PR TITLE
Fix assets for 2.3

### LINKED ISSUE #3285: Actual assets get deleted on assets:install
### Issue details

If I try to test the 2.3 branch of wallabag, the `php bin/console assets:install --env=prod --symlink --relative` Step deletes everything below `web/bundles/wallabagcore` which leaves the app in a pretty unuseable state.

### Environment

* wallabag version (or git revision) that exhibits the issue: 
  * revision f0cd12f
* How did you install wallabag? Via `git clone` or by downloading the package?
  * Git clone in a docker container. See https://github.com/janLo/wallabag-docker/tree/testing
* Last wallabag version that did not exhibit the issue (if applicable):
  * 2.2.x
* php version:
  * 7.x  (what is in alpine)
* OS:
  * alpine Linux
* type of hosting (shared or dedicated):
  * Container behind reverse proxy
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL):
  * Postgres

### Steps to reproduce/test case
Build the container from the link above.

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | -
| Documentation | no
| Translation   | no
| Fixed tickets |  #3285 
| License       | MIT

This PR resolves the issue with composer assets:install and assets in `web/bundles/wallabagcore` managed with `yarn`.

All assets managed with `yarn` have been moved to `web/wallassets`.

Tested on dev and prod. Let's test yourself too :-)

Fixes #3285

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,12 +23,11 @@
...
 web/uploads/
-!web/bundles
-web/bundles/*
-!web/bundles/wallabagcore
+/web/bundles/*
+!/web/bundles/.gitkeep
 /web/assets/images/*
 !web/assets/images/.gitkeep
-web/bundles/wallabagcore/*.dev.js
+/web/wallassets/*.dev.js
```
Comment (tcitworld) [inline .gitignore]: :+1:

**Comment by Kdecherf** [comment]
@tcitworld did you test it?

**Comment by tcitworld** [comment]
👍

**Comment by j0k3r** [comment]
:tada:

**Pull request merged**